homeassistant/components/homeassistant/scene.py

```diff
@@ -271,6 +282,25 @@ async def create_service(call: ServiceCall) -> None:
         SCENE_DOMAIN, SERVICE_CREATE, create_service, CREATE_SCENE_SCHEMA
     )
 
+    async def delete_service(call: ServiceCall) -> None:
+        """Delete a dynamically created scene."""
+        entity_id = call.data[CONF_ENTITY_ID]
```

Comment:
Why not use scene_id?

Reply:
We should not use `scene_id`. The parameter `scene_id` in the create service isn't an entity_id. It's the name of the service to be created. When configuring scenes in YAML, this parameter is actually called `name`. I don't think we would have named this parameter `scene_id` today in the create service.

`entity_id` is good as we want an entity_id to find the existing scene entity.